massa-models/src/block.rs

```diff
@@ -206,6 +206,16 @@ impl Serializer<Block> for BlockSerializer {
     }
 }
 
+///
```

Comment:
TODO DOc

Reply:
This needs a followup issue

Reply:
note: suppressing warnings by adding fake empty comments is not a good practice

Reply:
#3707